doc/jvm/xgboost4j_spark_tutorial.rst

```diff
@@ -290,7 +290,20 @@ When interacting with other language bindings, XGBoost also supports saving-mode
 
 * You can build XGBoost4J-Spark with the steps described in `here <https://xgboost.readthedocs.io/en/latest/jvm/index.html#installation-from-source>`_, but turning `USE_HDFS <https://github.com/dmlc/xgboost/blob/e939192978a0c152ad7b49b744630e99d54cffa8/jvm-packages/create_jni.py#L18>`_ (or USE_S3, etc. in the same place) switch on. With this approach, you can reuse the above code example by replacing "nativeModelPath" with a HDFS path.
 
-However, if you build with USE_HDFS, etc. you have to ensure that the involved shared object file, e.g. libhdfs.so, is put in the LIBRARY_PATH of your cluster. To avoid the complicated cluster environment configuration, we recommend download file manually from HDFS and load with the pre-built version of XGBoost.
+However, if you build with USE_HDFS, etc. you have to ensure that the involved shared object file, e.g. libhdfs.so, is put in the LIBRARY_PATH of your cluster. To avoid the complicated cluster environment configuration, we recommend the following steps to pass models with HDFS, S3, etc. (taking HDFS as an example)
+
+* create a new file with **val outputStream = fs.create("hdfs_path")**
+
+* pass the returned OutputStream in the first step to **xgbClassificationModel.nativeBooster.saveModel(outputStream)**
+
+* download file in other languages from HDFS and load with the pre-built (without the requirement of libhdfs.so) version of XGBoost:
+
+.. code-block:: python
+
+  import xgboost as xgb
+  bst = xgb.Booster({'nthread': 4})
+  local_path = download_from_hdfs("hdfs_path")
```

Comment:
In which package do we get `download_from_hdfs()`?

Reply:
no....it's a dummy helper function which is supposed to be implemented by the user